FishError/BlobRoss
# Feature/blb 438 red lore

Finished Red Lore

To test:
- play the game normally til you get to the sewer
- make sure you have all 4 red puzzle pieces (you can simply drag and drop them into the scene)
- talk to the lore master npc
- and click through the red lore
